README.rst

```diff
@@ -1,6 +1,8 @@
-django-parasol
+parasol
 ==============
 
+.. sphinx-start-marker-do-not-remove
+
 **django-parasol** is a reusable `Django`_ application to simplify interacting
```

Comment:
if we changed the name to just 'parasol', should it change on this line too?

Reply:
Absolutely should. I was planning to circle back to it after I did the branch merge, so thank you for flagging it up!